logpolicy/logpolicy.go

```diff
@@ -488,7 +495,12 @@ func NewWithConfigPath(collection, dir, cmdName string, netMon *netmon.Monitor)
 	if cmdName == "" {
 		cmdName = version.CmdName()
 	}
-	tryFixLogStateLocation(dir, cmdName)
+
+	haveCustomLogf := logf == nil
```

Comment:
Shouldn't this be `logf != nil`?

Reply:
Perhaps just invert this variable as "useStdLogger"? That way the conditionals below read nicer.

Reply:
Gooooood catch 🤦 The rename makes this much clearer.